server/webapp/WEB-INF/rails.new/config/routes.rb

```diff
@@ -240,6 +240,7 @@
         end
         post 'internal/security/auth_configs/verify_connection' => 'security/auth_configs#verify_connection', as: :internal_verify_connection
 
+        resources :config_repos, param: :id, only: [:create, :update, :show, :index, :destroy], constraints: {id: CONFIG_REPO_ID_FORMAT}
```

Comment:
The [default filter](https://github.com/gocd/gocd/blob/master/server/webapp/WEB-INF/applicationContext-acegi-security.xml#L296) will take care of this endpoint as it is only accessible by **admins**